src/background.js

```diff
@@ -342,20 +354,63 @@ const rollout = {
     // Determine to skip additional heuristics (by presence of an enterprise policy)
     if (policyEnableDoH === "no_policy_set") {
       // Resetting skipHeuristicsCheck in case a user had a policy and then removed it!
-      await rollout.setSetting("doh-rollout.skipHeuristicsCheck", false);
+      await rollout.setSetting(DOH_SKIP_HEURISTICS_PREF, false);
     } else {
       // Don't check for prefHasUserValue if policy is set to disable DoH
-      await rollout.setSetting("doh-rollout.skipHeuristicsCheck", true);
+      await rollout.setSetting(DOH_SKIP_HEURISTICS_PREF, true);
     }
     return;
 
   },
 
+  async migrateLocalStoragePrefs() {
+
+    // Migrate updated local storage item names. If this has already been done once, skip the migration
+    const isMigrated = await browser.experiments.preferences.getBoolPref(DOH_BALROG_MIGRATION_PREF, false);
+
+    if (isMigrated) {
+      return;
+    }
+
+    // Check all local items, and migrate them to prefs. If no value set, ignore them.
+    const localStorageItems = [
```

Comment:
nit: These are not actually local storage items, they're just keys. From that perspective, I'd suggest calling this `oldKeys` or something. Not a big deal though.

Reply:
Revised to `legacyLocalStorageKeys` and also added comment to explain what they are.